connectors/sources/azure_blob_storage.py

```diff
@@ -185,6 +185,7 @@ async def get_content(self, blob, timestamp=None, doit=None):
         if not self.can_file_be_downloaded(file_extension, filename, file_size):
             return
 
+        self._logger.debug(f"Downloading content for file: {filename}")
```

Comment:
Looks like a duplicate line - you already have "f"Downloading content for blob: {blob_name} from {container_name} container"" that happens right after